<launch>
    <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

    <node pkg="rover" type="simple_drive" name="simple_drive" clear_params="true" output="screen" respawn="false">
      <!-- INPUT TOPICS -->
      <remap from="cmd_vel" to="cmd_vel" />
      <remap from="servo_pos" to="servo_pos" />
    </node>
</launch>
